### PR TITLE
(Feature, CORE) Add ValidatorMetadata to drop-down list in "Modify Proxy Contract Ballot"

### PR DESCRIPTION
**Pull request by varasev**
**(Mandatory) Description**

These changes correspond to the new version of ProxyStorage contract from https://github.com/poanetwork/poa-network-consensus-contracts/pull/65 PR.

Also it includes small fixes described in https://github.com/poanetwork/poa-network-consensus-contracts/issues/45, https://github.com/poanetwork/poa-dapps-voting/issues/117 and https://github.com/poanetwork/poa-dapps-voting/issues/119.

**(Mandatory) What is it: (Fix), (Feature), or (Refactor)**

(Feature)

### CODE REVIEW

File: src/components/BallotProxyMetadata.jsx [hunk 1]
```diff
@@ -35,6 +35,8 @@ export class BallotProxyMetadata extends React.Component {
                   { value: '3', label: ballotStore.ProxyBallotType[3] },
                   { value: '4', label: ballotStore.ProxyBallotType[4] },
                   { value: '5', label: ballotStore.ProxyBallotType[5] },
+                  { value: '7', label: ballotStore.ProxyBallotType[7] },
```
Comment (vbaranov) [inline src/components/BallotProxyMetadata.jsx:38]: Why do we skip 6th index here?

File: src/stores/BallotStore.js [hunk 2]
```diff
@@ -22,7 +22,9 @@ class BallotStore {
 		2: 'VotingToChangeKeys',
 		3: 'VotingToChangeMinThreshold',
 		4: 'VotingToChangeProxy',
-		5: 'BallotsStorage'
+		5: 'BallotsStorage',
+		7: 'ValidatorMetadata',
```
Comment (vbaranov) [inline src/stores/BallotStore.js:26]: Why do we skip 6th index here?

**Comment by vbaranov** [comment]
@varasev maybe, let's add a solution for https://github.com/poanetwork/poa-dapps-voting/issues/117 bug in this PR?

File: src/components/BallotProxyMetadata.jsx (hunk 1 above)
Comment (varasev) [reply src/components/BallotProxyMetadata.jsx:38]: I took indexes from ProxyStorage.sol:

```
enum ContractTypes {
    Invalid,
    KeysManager,
    VotingToChangeKeys,
    VotingToChangeMinThreshold,
    VotingToChangeProxy,
    BallotsStorage, 
    PoaConsensus,
    ValidatorMetadata,
    ValidatorMetadataEternalStorage
}
```

i.e. 6th index is for `PoaConsensus`.

File: src/stores/BallotStore.js (hunk 2 above)
Comment (varasev) [reply src/stores/BallotStore.js:26]: I took indexes from ProxyStorage.sol:

```
enum ContractTypes {
    Invalid,
    KeysManager,
    VotingToChangeKeys,
    VotingToChangeMinThreshold,
    VotingToChangeProxy,
    BallotsStorage, 
    PoaConsensus,
    ValidatorMetadata,
    ValidatorMetadataEternalStorage
}
```

i.e. 6th index is for `PoaConsensus`.

**Comment by varasev** [comment]
Sure, I'll do.

**Review by vbaranov — APPROVED** [review]

**Pull request merged**